builder/bundleset.go

```diff
@@ -0,0 +1,188 @@
+package builder
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var (
+	validBundleNameRegex  = regexp.MustCompile(`^[A-Za-z0-9-_]+$`)
+	validPackageNameRegex = regexp.MustCompile(`^[A-Za-z0-9-_+.]+$`)
+)
+
+type bundle struct {
+	Name     string
+	Filename string
+
+	DirectIncludes []string
```

Comment:
I was thinking about someone reading code that says "bundle.Includes", and there could be ambiguity if this were all the includes or just the direct ones. Since this notion of Direct/All already appears for package I just reused.